test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+# set path for test directory
+TESTS_PATH=./test
+
+# setup to run browser GUI
+export DISPLAY=:99.0
+sh -e /etc/init.d/xvfb start &
+sleep 3 # give xvfb some time to start
+
+# run local server in the background and wait until it starts
+# ref: https://stackoverflow.com/a/21002153
+echo "Running local server..."
+exec 3< <(npm run dev)
+sed '/webpack: Compiled successfully.$/q' <&3 ; cat <&3 &
+
+# go to test directory
+cd $TESTS_PATH
+
+# install gem dependencies using monster_rsa private key to fetch the monster gem
+GIT_SSH_COMMAND="ssh -i ~/.ssh/monster_rsa" bundle install
```

Comment:
Out of interest what version of ruby are we using in travis?

Reply:
We're using a default one that's installed on Travis. Just checked and apparently it's quite an old version we should consider updating:
`ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]`